src/commands/slowmode_commands.py

```diff
@@ -0,0 +1,31 @@
+import discord
+import logging
+from discord.ext.commands import Bot
+from settings import get_settings
+from services.slowmode_service import edit_slowmode
+from util import is_user_moderator
+from .helper import create_logging_embed, create_response_context
+
+settings = get_settings()
+logger = logging.getLogger(__name__)
+
+
+def create_slowmode_commands(bot: Bot) -> None:
+    @bot.tree.command()
+    @discord.app_commands.check(is_user_moderator)
+    @discord.app_commands.describe(
+        channel="Channel name slowmode is being modified in",
+        interval="The number of seconds desired between each message. Must be between 0 (off) and 21600 (6 hours).",
+    )
+    async def set_slowmode(
+        interaction: discord.Interaction,
+        interval: int,
+        channel: discord.TextChannel = None,
```

Comment:
i think maybe we should have it the other way since it's more intuitive to set the channel and then the interval